test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationSampleTest.cs

```diff
@@ -11,14 +11,11 @@ namespace Microsoft.AspNetCore.Localization.FunctionalTests
 {
     public class LocalizationSampleTest
     {
-        private static readonly string _applicationPath =  Path.Combine("samples", "LocalizationSample");
+        private static readonly string _applicationPath = Path.Combine("samples", "LocalizationSample");
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux)]
-        [OSSkipCondition(OperatingSystems.MacOSX)]
-        [InlineData(RuntimeFlavor.Clr, "http://localhost:5080/", RuntimeArchitecture.x64)]
+        [Theory]
         [InlineData(RuntimeFlavor.CoreClr, "http://localhost:5081/", RuntimeArchitecture.x64)]
```

Comment:
`[Theory]`s that run once are just strange. Later in this PR you've moved from the more common `[Fact]` with hard-coded flavour, URL and architecture to this style. I suggest standardizing on the opposite i.e. favouring `[Fact]`s or `[ConditionalFact]`s where possible.